src/main/resources/model/Locador.java

```diff
@@ -0,0 +1,14 @@
+/*

```

Comment:
@MilenaMaced Todo arquivo deve começar com a licença escolhida para o projeto.

Reply:
@fabriciofx Devo colocar a licença na IDE?

Reply:
@MilenaMaced Crie um arquivo LICENSE.txt na raiz do seu projeto que contenha a licença e configure a sua IDE para colocar o conteúdo desse arquivo no início de cada arquivo.

Reply:
@fabriciofx Adicionei a Licença, está correto a forma que foi feita?